chapter_01.asciidoc

```diff
@@ -241,10 +248,17 @@ TIP: If you see an error saying "no module named selenium",
 Well, it may not look like much, but that was our first ever passing test!
 Hooray!
 
+// DAVID: Not for me I'm afraid.
+// assert "Django" in browser.title
+//           ^^^^^^^^^^^^^^^^^^^^^^^^^
+// AssertionError
+// The page title I get, on Django 4.2.1, is <title>The install worked successfully! Congratulations!</title>
```

Comment:
ah thanks , @Xronophobe mentioned this too.  i must fix the tests for this chapter, 'cos they shouldn't have let me get away with this...